staging/src/k8s.io/apiserver/pkg/server/routes/profiling.go

```diff
@@ -26,8 +27,9 @@ import (
 type Profiling struct{}
 
 // Install adds the Profiling webservice to the given mux.
-func (d Profiling) Install(c *mux.APIContainer) {
+func (d Profiling) Install(c *mux.APIContainer, secureConnHandler func(http.ResponseWriter, *http.Request)) {
```

Comment:
Agreed. Should probably be broken out of profiling as well (into something like debug) and given its own control flag.